torchvision/datasets/omniglot.py

```diff
@@ -21,8 +21,9 @@ class Omniglot(VisionDataset):
             puts it in root directory. If the zip files are already downloaded, they are not
             downloaded again.
     """
+
     folder = 'omniglot-py'
-    download_url_prefix = 'https://github.com/brendenlake/omniglot/raw/master/python'
+    download_url_prefix = 'https://raw.githubusercontent.com/brendenlake/omniglot/master/python/'
```

Comment:
Together with L95 this would result in `//`.

```suggestion
    download_url_prefix = 'https://raw.githubusercontent.com/brendenlake/omniglot/master/python'
```

Reply:
Nice catch. :sweat_smile: